client/web/compose/src/components/PageBlocks/Navigation/NavTypes/Url.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <tr>
+    <td />
+    <td>
+      <b-form-group :label="$t('navigation.fieldLabel')">
+        <b-form-input
+          v-model="value.options.itemOption.label"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.url')">
+        <b-form-input
+          v-model="value.options.itemOption.url"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.openIn')">
+        <b-form-select
+          v-model="value.options.itemOption.targetType"
+          :options="openInType"
+          size="sm"
+        >
+          <template #first>
+            <b-form-select-option :value="undefined">
+              {{ $t("navigation.none") }}
+            </b-form-select-option>
+          </template>
+        </b-form-select>
+      </b-form-group>
+    </td>
+  </tr>
+</template>
+
+<script>
+export default {
+  i18nOptions: {
+    namespaces: 'block',
+  },
+
+  props: {
+    value: {
+      type: Object,
+      required: true,
+    },
+  },
+
+  data () {
+    return {
+      openInType: [
+        { value: '_self', text: this.$t('navigation.sameTab') },
+        { value: '_blank', text: this.$t('navigation.newTab') },
```

Comment:
Since the two values are the same, if I save new window it thinks I saved newTab. So you'll need something like Atanas did in record-list for the `recordDisplayOption`

Reply:
the value for new window and new tab is the same for the `target` value, I don't seem to know why. Atanas implementation does not have a new window

Reply:
True, but that doesnt change the fact that your values shouldn't be _self and _blank, but sameTab, newTab, newWindow